src/App.js

```diff
@@ -1,12 +1,21 @@
-import React from 'react';
+import React, { Component } from "react";
+import DisplayAllArticles from "./Components/DisplayAllArticles";
+import {Container, Header,Divider} from 'semantic-ui-react';
 
-
-function App() {
-  return (
-    <div>
-     <h1>GNOSIS</h1>
-    </div>
-  );
+class App extends Component {
+  render() {
+    return (
+      <Container>
+        <Header as='h1' textAlign='center'>
+          GNOSIS
+      </Header>
+      <Divider/>
+        <>
```

Comment:
Why are you wrapping`<DisplayAllArticles />` in a fragment?

Reply:
The `fragment` tags are a hangover from a commit before the `Container` tags were added.  The `fragment` tags have been removed.